smo-backend/src/index.ts

```diff
@@ -32,10 +34,17 @@ import msgpackParser from "socket.io-msgpack-parser";
 import cors from "cors";
 import express from "express";
 
+function buildHttpsServer() {
+  const cert = readFileSync(process.env.CERTFILE as string);
+  const key = readFileSync(process.env.KEYFILE as string);
+  return https.createServer({ key, cert }, app);
```

Comment:
`app` doesn't exist here, it will be a runtime error, I think you forgot to pass it as a parameter.

Reply:
In order to resolve this, `http.RequestListener<typeof http.IncomingMessage, typeof http.ServerResponse>` as parameter type is used.